### PR TITLE
[12.0][MIG] iot_template_oca

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
I did not rename the module to make easier the migration later on

ping @ForgeFlow
ping @etobella 
ping @lubusax

Depends on:
- [x] https://github.com/OCA/iot/pull/55

### CODE REVIEW

File: iot_template_oca/views/iot_template_views.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            <tree>
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="iot_template_action">
+        <field name="name">IoT Templates</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">iot.template</field>
+        <field name="view_mode">tree,form</field>
```
Comment (MiquelRForgeFlow) [inline iot_template_oca/views/iot_template_views.xml:100]: add view_type

Comment (AaronHForgeFlow) [reply iot_template_oca/views/iot_template_views.xml:100]: Nice catch, changes requested applied now

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by lubusax — APPROVED** [review]
Funcional tests on a RAS device OK

**Pull request merged**